Source/Cvent.SchemaToPoco.Core/JsonSchemaResolver.cs

```diff
@@ -115,19 +117,26 @@ private JsonSchemaWrapper ResolveSchemaHelper(Uri curr, Uri parent, string data)
             {
                 // Get the full path to the file, and change the reference to match
                 string refName = match.Groups[1].Value;
-                if (!refName.EndsWith(".json"))
-                {
-                    refName += ".json";
+                var currPath = new Uri(refName, UriKind.RelativeOrAbsolute);

```

Comment:
It could, though IDs are supposed to be valid URIs so in practice it's fine where it is. Given that changing anything here involves hitting 3 repositories (this one and checking the binary in to Engine and the .NET client), I'd rather leave this as is since it's not really _wrong_, it just could be better.

I suppose an alternative would be to make the code better here but not update the binaries since we don't expect a functional difference, but that seems messy.